api/actions/v1alpha1/filters_types.go

```diff
@@ -0,0 +1,27 @@
+package v1alpha1
+
+import "github.com/odigos-io/odigos/api/k8sconsts"
+
+const ActionNameFilters = "Filters"
```

Comment:
the name "filters" is very generic, and (especially in ui) can be quite ambiguous.

consider other names that have better association for normal odigos user which is not familiar with opentelemetry collector terminology.